build.gradle

```diff
@@ -16,8 +16,8 @@ subprojects {
     }
 
     java {
-        sourceCompatibility = JavaVersion.VERSION_1_6
-        targetCompatibility = JavaVersion.VERSION_1_6
+        sourceCompatibility = JavaVersion.VERSION_1_8
```

Comment:
Is Java 1.8 necessary to compile argon2 now? if not, i'd prefer to stay on 1.6. AFAIK android builds break if this is changed > 1.6

Reply:
I will revert it back to 1.6.

I figured out it was just my IDE internal gradle plugin.
When using gradle directly or `./gradlew` it works just fine.